File: Java/src/br/com/dominio/s03/Exe02.java
package br.com.dominio.s03;

import java.util.Scanner;

public class Exe02 {

	public static void main(String[] args) {
		//variaveis
		int quantidade_minima, quantidade_maxima, soma;
		float media;
		Scanner teclado = new Scanner(System.in);
		
		//entradas
		System.out.println("Informe a quantidade mímina do estoque: ");
		quantidade_minima = teclado.nextInt();
		System.out.println("Informe a quantidade máxima do estoque: ");
		quantidade_maxima = teclado.nextInt();
		
		//processamento
		soma = quantidade_minima + quantidade_maxima;
		media = soma / 2f; // uma das variaveis aqui tem que estar com ponto flutuante "f" se não não sai o resultado correto
		
		//saida
		System.out.println("O estoque médio é: " +media);
		
		teclado.close();
	}

}

File: Python/Seção03/Exe02.py
#Entradas
quant_min = int(input("Informe a quantidade mínima para o estoque: "))
quant_max = int(input("Informe a quantidade máxima para o estoque: "))

#Processamento
estoque_medio = (quant_min + quant_max) / 2

#saída
print("O estoque médio é de {0}".format(estoque_medio))
